content/05.conclusions.md

```diff
@@ -3,4 +3,53 @@
 We implemented AI-based models into publishing infrastructure.
 While most manuscripts have been written by humans, the process is time consuming and academic writing can be difficult to parse.
 We sought to develop a technology that academics could use to make their writing more understandable without changing the fundamental meaning.
-This work lays the foundation for a future where academic manuscripts are constructed by a process that incorporates both human and machine authors.
\ No newline at end of file
+This work lays the foundation for a future where academic manuscripts are constructed by a process that incorporates both human and machine authors.
+
+
+We found that, in general, the tool generated reasonable suggestions for most sections of the manuscripts, with paragraph that were greatly improved and others that needed minimal human review.
+We also found, though, that for some paragraph the revision was not good enough, and in some cases it even made the text worse.
+This was particularly true for short paragraphs and some in the the Methods sections when mathematical expressions were present.
+
+Abstracts:
+
+  - remember that the models are stochastic (meaning that we can always run it again).
+  - for the phenoplier abstract, the revision was not really great (the AI model removed too much background information).
+    - the user of the tool, in these cases, might want to run it again to obtain another revision.
+    - for abstract, maybe we need to improve the prompts to say something like "keep more background information".
+
+
+Introductions:
+
+  - really good for both manuscripts
+  - in phenoplier, the models did not keep the format of citations for one paragraph, although this can be easily fixed by a human.
+    - the other paragraph that did not converge was a large one; this could be an indication that is not easy to understand and it should be split into smaller paragraphs.
+    - therefore, the fact that the tool cannot process a paragraph of text can also provide insight about the quality of the writing.
+
+
+Methods:
+  - the model not only improved the text, but also fixed references to wrong mathematical symbols.
+
+
+One way to fix the problem of context is to use chatbots like ChatGPT, which could enable the same paragraph-by-paragraph approach to revision that we presented here, but with higher quality revisions by considering the context (just as if it were reading it).
+  - This approach (chatGPT, etc) has also the advantage of being independent of the context length of the models.
+  - Here, mention this kind of models in general again, besides ChatGPT, like AnthropicAI's Claude model (a competitor of ChatGPT): https://twitter.com/goodside/status/1611556749726605312
+
+
+Edits vs completion endpoints:
```

Comment:
This is probably a sentence at most, perhaps in a single paragraph combined with the above points.